x-pack/plugins/enterprise_search/public/applications/shared/layout/nav.tsx

```diff
@@ -275,16 +269,14 @@ export const useEnterpriseSearchAnalyticsNav = (
   }
 ) => {
   const navItems = useEnterpriseSearchNav();
-  const collectionNav = navItems.find(
-    (item) =>
-      item.id === 'enterpriseSearchAnalytics' && item.items?.[0]?.id === 'analytics_collections'
-  )?.items?.[0];
+  const applicationsNav = navItems.find((item) => item.id === 'applications');
+  const analyticsNav = applicationsNav?.items?.find((item) => item.id === 'analytics_collections');
 
-  if (!name || !paths || !collectionNav) return navItems;
+  if (!name || !paths || !analyticsNav) return navItems;
 
-  collectionNav.items = [
+  analyticsNav.items = [
     {
-      id: 'analytics_collections',
+      id: 'analytics_collection',
       items: [
         {
           id: 'enterpriseSearchEngineOverview',
```

Comment:
not a fault of your PR but noticed our ids are a copy & paste from engines. Will fix if you dont.

Reply:
I'll fix while I'm here